cardano-ping/src/Cardano/Network/Ping.hs

```diff
@@ -539,6 +575,30 @@ pingClient stdout stderr PingOpts{pingOptsQuiet, pingOptsJson, pingOptsCount, pi
           [] -> Left $ "No overlapping versions with " <> show versions
           vs -> Right $ foldr1 max vs
 
+    showNetworkRtt :: Double -> String
+    showNetworkRtt rtt =
+      if pingOptsJson
+        then printf "{\"network_rtt\":%.3f}" rtt
```

Comment:
We should use something like: `encode $ Map.fromList [("network_rtt", rtt)]`. Note that this returns a lazy `ByteString`, so we'd need to write it with `hPutStrLn stdout` using an appropriate `hPutStrLn` from the `ByteString` library.

Reply:
Similar for the other things below.

Reply:
Yes you're right, this was a tad unsafe.

I changed to `encode $ object ["network_rtt" .= toJSON rtt]` (using `object` to avoid `Map.fromList` which would require adding `containers` as a dependency).

I wrote to stdout using `putStr` + `<> "\n"` because there is [no `Ln` version of `put` for `Data.ByteString.Lazy`](https://hoogle.haskell.org/?hoogle=putStrLn%20package%3Abytestring) :+1: